src/simcore_service_deployment_agent/auto_deploy_task.py

```diff
@@ -433,9 +433,11 @@ async def auto_deploy(app: web.Application):
         app["state"][TASK_NAME] = State.STOPPED
         return
     except TagSyncErrorException:
-        log.warning(
-            "Problem  while initializing deployment: Tag-Sync specified but latest tags did not match. Continuing polling..."
+        log.error(
+            "Problem while initializing deployment: Tag-Sync specified but latest tags did not match. Continuing polling..."
```

Comment:
I guess that if the application now crashes on start, the message shall be changed...

Reply:
I will address this